radicle-cli/src/commands/issue.rs

```diff
@@ -245,3 +299,17 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+fn show_issue(issue: &issue::Issue) -> anyhow::Result<()> {
+    println!("title: {}", issue.title());
+    println!("state: {}", issue.state());
+
+    let tags: Vec<String> = issue.tags().cloned().map(|t| t.into()).collect();
+    println!("tags: {}", tags.join(", "));
+
+    let assignees: Vec<String> = issue.assigned().map(|a| a.to_string()).collect();
+    println!("assignees: {}", assignees.join(", "));
+
+    println!("{}", issue.description().unwrap_or(""));
```

Comment:
Should these be `terminal::info`?

Reply:
Updated to use info(..), but not entirely sure when and when it should be preferred.